packages/react/src/components/TimePicker/TimePicker-test.js

```diff
@@ -20,8 +20,11 @@ describe('TimePicker', () => {
     });
 
     it('passes classNames as expected', () => {
-      render(<TimePicker id="time-picker" className="🚀" />);
-      expect(screen.getByRole('textbox')).toHaveClass('🚀');
+      const { container } = render(
+        <TimePicker id="time-picker" className="🚀" />
+      );
+      expect(container.querySelector('.🚀')).toBeInTheDocument();
+      expect(screen.getByRole('textbox')).not.toHaveClass('🚀');
```

Comment:
Just to avoid unicode parsing issues and keep this closer to real-world usage:

```suggestion
        <TimePicker id="time-picker" className="test-class" />
      );
      expect(container.querySelector('.test-class')).toBeInTheDocument();
      expect(screen.getByRole('textbox')).not.toHaveClass('test-class');
```

Reply:
@tay1orjones Thank you for the feedback. I have updated the code.